### PR TITLE
Update to include newest sloctl features

### PR DESCRIPTION
**Pull request by greg-agacinski**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,23 +8,50 @@ This action applies a Nobl9 configuration file (specified via the sloctl_yml inp
...
+```yaml
+name: Nobl9 GitHub Actions Demo
+on: [push]
+jobs:
+  nobl9:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v2
+      - uses: nobl9/nobl9-action@v0.2.1
```
Comment (nieomylnieja) [inline README.md]:
```suggestion
      - uses: nobl9/nobl9-action@v0.2.2
```

File: entrypoint.sh [hunk 2]
```diff
@@ -9,8 +9,15 @@ defaultContext = "default"
...
     clientId = "${INPUT_CLIENT_ID}"
     clientSecret = "${INPUT_CLIENT_SECRET}"
-    accessToken = "${INPUT_ACCESS_TOKEN}"
-    project = "${INPUT_PROJECT}"
 EOF
 
-sloctl apply -f "${INPUT_SLOCTL_YML}"
+flags=(
+  -f "${INPUT_SLOCTL_YML}"
+  -y # default to auto confirm
```
Comment (nieomylnieja) [inline entrypoint.sh]: Let's maybe leave some context here:

```suggestion
  -y # Required to auto confirm, for more details refer to: https://docs.nobl9.com/sloctl-user-guide?_highlight=prompt&_highlight=threshold#apply
```

File: README.md [hunk 3]
```diff
@@ -8,23 +8,50 @@ This action applies a Nobl9 configuration file (specified via the sloctl_yml inp
...
 ## Inputs
 
-### `client_id`
-The Client ID of your Nobl9 account
+| Parameter | Description | Required | Default |
+| --- | --- | --- | --- |
+| `client_id` | The Client ID of your Nobl9 account | **Yes** | N/A |
+| `client_secret` | The Client Secret of your Nobl9 account | **Yes** | N/A |
+| `sloctl_yml` | The path or [glob pattern](https://pkg.go.dev/path/filepath#Match) to the configuration in YAML format, relative to the root directory of the repository | **Yes** | N/A |
+| `dry_run` | The path or glob pattern to the configuration in YAML format, relative to the root directory of the repository | No | `false` |
```
Comment (nieomylnieja) [inline README.md]: Wrong description ;)

**Review by nieomylnieja — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (greg-agacinski) [inline README.md]:
```suggestion
| `dry_run` | Submits server-side request without persisting the configured resources | No | `false` |
```

**Review by nieomylnieja — DISMISSED** [review]
lgtm, didn't test it though

**Review by nieomylnieja — APPROVED** [review]

**Pull request merged**